### PR TITLE
[EPA FLIGHTS] Processing script for annual data

### PR DESCRIPTION
**Pull request by beets**
* Generate schema for new classes and stat vars per discussion
* Generate flattened CSV for _most_ sheets
* Validated import
* Added tests

### CODE REVIEW

File: scripts/us_epa/ghgrp/import_data/schema.mcf [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Node: dcs:GreenhouseGas
+typeOf: schema:Class
+subClassOf: dcs:Enumeration
```
Comment (pradh) [inline scripts/us_epa/ghgrp/import_data/schema.mcf:3]: Should this be a subClassOf GasType ?

File: scripts/us_epa/ghgrp/import_data/schema.mcf [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+description: "Gases that trap heat in the atmosphere."
+descriptionUrl: "https://www.epa.gov/ghgemissions/overview-greenhouse-gases"
+
+Node: dcs:EmissionSourceEnum
+typeOf: schema:Class
```
Comment (pradh) [inline scripts/us_epa/ghgrp/import_data/schema.mcf:8]: Prod may still complain about requiring names for the schema things... I think.

File: scripts/us_epa/ghgrp/gas_test.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            'Nitrous Oxide (N2O) emissions':
+                'Annual_Emissions_NitrousOxide',
+            'HFC emissions':
+                'Annual_Emissions_HFC',
+            'PFC emissions':
+                'Annual_Emissions_PFC',
+            'SF6 emissions':
+                'Annual_Emissions_SF6',
+            'NF3 emissions':
+                'Annual_Emissions_NF3',
```
Comment (pradh) [inline scripts/us_epa/ghgrp/gas_test.py:46]: Should we expand HFC, PFC, HFE ?  Esp if the enums are just called HFC

File: scripts/us_epa/ghgrp/gas.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        if col.startswith('Total reported') or col.startswith('Biogenic'):
+            fp.write(f'emissionSourceType: dcs:NonBiogenicEmissionSource')
+        if 'non-biogenic' in col:
+            fp.write(f'emissionSourceType: dcs:BiogenicEmissionSource')
+
+
+if __name__ == '__main__':
+    with open(os.path.join('import_data', 'gas.mcf'), 'w') as fp:
+        append_gas_mcf(fp)
+        append_sv_mcf(fp)
```
Comment (pradh) [inline scripts/us_epa/ghgrp/gas.py:134]: nit: fine if we are going to manually separate them, but the gas enums and svs will into separate paths in schema repo right?

**Review by pradh — APPROVED** [review]
Very nice!

File: scripts/us_epa/ghgrp/gas.py (hunk 4 above)
Comment (beets) [reply scripts/us_epa/ghgrp/gas.py:134]: yes. good point - i'll write these to different files

File: scripts/us_epa/ghgrp/gas_test.py (hunk 3 above)
Comment (beets) [reply scripts/us_epa/ghgrp/gas_test.py:46]: oh thanks for catching that. yes should have the gas names spelled out

File: scripts/us_epa/ghgrp/import_data/schema.mcf (hunk 2 above)
Comment (beets) [reply scripts/us_epa/ghgrp/import_data/schema.mcf:8]: i'll add names here and elsewhere just to be on the safe side

File: scripts/us_epa/ghgrp/import_data/schema.mcf (hunk 1 above)
Comment (beets) [reply scripts/us_epa/ghgrp/import_data/schema.mcf:3]: thanks - good catch

**Comment by beets** [comment]
thanks for the review!

**Pull request merged**